fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -477,13 +478,17 @@ Future<Optional<std::string>> createMetacluster(Reference<DB> db, ClusterName na
 				}
 			}
 
+			wait(managementClusterCheckEmpty(tr));
```

Comment:
Was this moved to provide a better error if we are using this on an existing cluster? I think that's a good idea, the one thing I'll say is that previously this was reading the metacluster registration and checking the cluster emptiness concurrently. We should keep that property, but now we'll wait on the metacluster registration first instead.

Reply:
sounds good ill make this call concurrently